modules/openy_features/openy_demo_content/modules/openy_demo_nprogram/openy_demo_nprogram.install

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * OpenY Demo Node Program install file.
+ */
+
+/**
+ * Implements hook_uninstall().
+ */
+function openy_demo_nprogram_uninstall() {
```

Comment:
these are equal across all modules, could we extend our Uninstall Service with automated hook_uninstall() handling too? @hamrant @anprok

Reply:
@anprok `\Drupal::service('openy.modules_manager')->postUninstall($module_name);` already executed after module uninstall.

I think we need to move `destroyMigrationData` function to `openy.modules_manager` service. And provide logic here for detecting that uninstalled module related to demo content and in this case run `destroyMigrationData` function.

As result - you can remove all this `hook_uninstalls` form PR.

cc @podarok

Reply:
I think we can run `destroyMigrationData` every time, this method just should check is there anything to destroy or not. It should not be limited to the demo content only,if other modules will have migration configs this method should handle them as a well